I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 710, in script
    $ eir ("Soooo much, you down to hear it?")
  File "game/script.rpy", line 710, in <module>
    $ eir ("Soooo much, you down to hear it?")
  File "game/phone.rpy", line 305, in eir
    renpy.pause()
  File "game/phone.rpy", line 257, in execute
    screen txt_msg():
  File "game/phone.rpy", line 257, in execute
    screen txt_msg():
  File "game/phone.rpy", line 265, in execute
    window:
  File "game/phone.rpy", line 270, in execute
    vbox:
Exception: Style u'phonemenu' does not exist.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 710, in script
    $ eir ("Soooo much, you down to hear it?")
  File "/Applications/renpy-6.99.14.3-sdk/renpy/ast.py", line 912, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Applications/renpy-6.99.14.3-sdk/renpy/python.py", line 2004, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script.rpy", line 710, in <module>
    $ eir ("Soooo much, you down to hear it?")
  File "game/phone.rpy", line 305, in eir
    renpy.pause()
  File "/Applications/renpy-6.99.14.3-sdk/renpy/exports.py", line 1471, in pause
    rv = renpy.ui.interact(mouse='pause', type='pause', roll_forward=roll_forward)
  File "/Applications/renpy-6.99.14.3-sdk/renpy/ui.py", line 295, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/Applications/renpy-6.99.14.3-sdk/renpy/display/core.py", line 2699, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "/Applications/renpy-6.99.14.3-sdk/renpy/display/core.py", line 3091, in interact_core
    root_widget.visit_all(lambda i : i.per_interact())
  File "/Applications/renpy-6.99.14.3-sdk/renpy/display/core.py", line 541, in visit_all
    d.visit_all(callback, seen)
  File "/Applications/renpy-6.99.14.3-sdk/renpy/display/core.py", line 541, in visit_all
    d.visit_all(callback, seen)
  File "/Applications/renpy-6.99.14.3-sdk/renpy/display/core.py", line 541, in visit_all
    d.visit_all(callback, seen)
  File "/Applications/renpy-6.99.14.3-sdk/renpy/display/screen.py", line 430, in visit_all
    callback(self)
  File "/Applications/renpy-6.99.14.3-sdk/renpy/display/core.py", line 3091, in <lambda>
    root_widget.visit_all(lambda i : i.per_interact())
  File "/Applications/renpy-6.99.14.3-sdk/renpy/display/screen.py", line 440, in per_interact
    self.update()
  File "/Applications/renpy-6.99.14.3-sdk/renpy/display/screen.py", line 625, in update
    self.screen.function(**self.scope)
  File "game/phone.rpy", line 257, in execute
    screen txt_msg():
  File "game/phone.rpy", line 257, in execute
    screen txt_msg():
  File "game/phone.rpy", line 265, in execute
    window:
  File "game/phone.rpy", line 270, in execute
    vbox:
  File "/Applications/renpy-6.99.14.3-sdk/renpy/display/layout.py", line 524, in __init__
    self._clipping = self.style.clipping
  File "gen/styleclass.pxi", line 189, in renpy.styledata.styleclass.Style.clipping.__get__
  File "style.pyx", line 493, in renpy.style.StyleCore._get
  File "style.pyx", line 679, in renpy.style.build_style
  File "style.pyx", line 142, in renpy.style.get_full_style
  File "style.pyx", line 83, in renpy.style.get_style
Exception: Style u'phonemenu' does not exist.

Darwin-16.6.0-x86_64-i386-64bit
Ren'Py 7.3.2.320
Hurricane Like Me Renpy 7.2.2.491
Sun Sep 29 11:20:13 2019
